stable/memcached/templates/statefulset.yaml

```diff
@@ -119,3 +119,7 @@ spec:
       affinity:
 {{ toYaml . | trim | indent 8}}
 {{- end }}
+{{- with .Values.updateStrategy }}
```

Comment:
Looks like other charts have opted for setting specific keys here. Like `updateStrategy.type`. Should probably do the same here

Reply:
Gotcha, thanks for the suggestion